### PR TITLE
[stable/mariadb] Add nodeselectors for master and slave statefulsets

### PR DESCRIPTION
**Pull request by mikkelhegn**
#### What this PR does / why we need it:
This PR adds value options to define a nodeSelector for the master and slave statefulsets for mariadb.

#### Which issue this PR fixes
  - addresses #1899

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/mariadb/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 5.11.1
+version: 5.11.2
```
Comment (davidkarlsen) [inline stable/mariadb/Chart.yaml]:
```suggestion
version: 5.12.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/mariadb/Chart.yaml (hunk 1 above)
Comment (mikkelhegn) [reply stable/mariadb/Chart.yaml]: Oh - I need a signed commit - let me go create that...

**Comment by MikkelHegn** [comment]
Please let me know if I need to go back and add a sign-off to the commit missing, there are no changes in the commit after.

**Comment by juan131** [comment]
Hi @MikkelHegn 

You need to sign-off every single commit, even when you just rebase your branch from master. It's mandatory for the DCO tests.

**Comment by MikkelHegn** [comment]
I don't get how the check is still failing. The latest commit does has this as part of the commit message 

> Signed-off-by: Mikkel Mørk Hegnhøj <mikhegn@microsoft.com>

https://github.com/helm/charts/commit/82b47b159742acdedd01da6bd05e431edd39d7d3

**Comment by juan131** [comment]
Hi @MikkelHegn 

If you're struggling with the "sign-off", I recommend you to close this PR. Then you can update your master branch, create a new branch with the latest changes and ensure this time you sign every commit. Once you do this, create a new PR.

**Comment by MikkelHegn** [comment]
https://github.com/helm/charts/pull/13486

**Pull request closed without merging**